scripts/systemd-test-setup.bash

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+docker compose exec client cat firezone-linux-client > firezone-linux-client
+chmod u+x firezone-linux-client
+
+sudo cp scripts/firezone-client.service /etc/systemd/system/
```

Comment:
Does this not need a `systemctl daemon-reload`?

Reply:
AIUI, if you copy in a new service file and then do commands on it, systemd will fault it in. But if you run commands on an existing file (or maybe any file) and then change the file later, that does need a daemon-reload